docker/docker/src/main/java/org/arquillian/cube/docker/impl/client/containerobject/AfterContainerObjectObserver.java

```diff
@@ -0,0 +1,46 @@
+package org.arquillian.cube.docker.impl.client.containerobject;
+
+import org.arquillian.cube.CubeController;
+import org.arquillian.cube.containerobject.ConnectionMode;
+import org.arquillian.cube.spi.Cube;
+import org.arquillian.cube.spi.CubeRegistry;
+import org.arquillian.cube.spi.metadata.IsContainerObject;
+import org.jboss.arquillian.core.api.Instance;
+import org.jboss.arquillian.core.api.annotation.Inject;
+import org.jboss.arquillian.core.api.annotation.Observes;
+import org.jboss.arquillian.test.spi.event.suite.After;
+
+import java.util.List;
+import java.util.logging.Logger;
+
+/**
+ * Observer that stops and removes started container objects at the end of test method execution.
+ */
+public class AfterContainerObjectObserver {
+
+    private static final Logger logger = Logger.getLogger(AfterContainerObjectObserver.class.getName());
+
+    @Inject
+    Instance<CubeRegistry> cubeRegistryInstance;
+    @Inject Instance<CubeController> cubeControllerInstance;
```

Comment:
minor detail but either we break lines or we dont - otherwise it's a bit unreadable.